authplugin/src/main/java/io/pravega/auth/AuthHandler.java

```diff
@@ -9,16 +9,22 @@
  */
 package io.pravega.auth;
 
-import java.util.Map;
-
 /**
  * Custom authorization/authentication handlers implement this interface.
  * The implementations are loaded from the classpath using `ServiceLoader` (https://docs.oracle.com/javase/7/docs/api/java/util/ServiceLoader.html)
  * Pravega controller also implements this interface through {@link #PasswordAuthHandler}.
  *
- * Each custom auth handler is registered with a unique name.
- * A client selects its auth handler by setting a grpc header with a name "method". T
- * his is done by implementing `PravegaCredentials` interface and passing it to client calls.
+ * Each custom auth handler is registered with a unique name identifying a supported authentication scheme.
+ *
+ * The client supplies authentication credentials formatted as per HTTP 1.1 (RFC 7235):
+ * <pre>
+ *     Authentication: &lt;scheme&gt; &lt;token&gt;
```

Comment:
Fix special characters, please.

Reply:
Did I botch the markup?   I was trying to convey:
```
Authorization: <scheme> <token>
```

Reply:
yeah, maybe it is my bad, but I've been asking contributors to remove markup unless strictly necessary. What you have will render correctly.